packages/component-lib/src/components/block/AssetTitle.tsx

```diff
@@ -56,6 +58,10 @@ export const AssetTitle = withTranslation('common')(({
             </Typography>
         </Grid>
         <ButtonListRightStyled item xs={5} display={'flex'} flexDirection={'row'} justifyContent={'flex-end'}>
+            {/* <Button variant={'outlined'} size={'medium'} color={'primary'} style={{ minWidth: 120, textTransform: 'none' }}
+                onClick={showRamp}>
+                {t('labelAssetsBtnRamp')}
+            </Button> */}
```

Comment:
Remove commented elements that were used for testing as these can end up here forever